Common/GPU/Vulkan/VulkanLoader.cpp

```diff
@@ -309,7 +309,7 @@ void VulkanSetAvailable(bool available) {
 
 bool VulkanMayBeAvailable() {
 	// Unsupported in VR at the moment
-	if (IsVRBuild()) {
+	if (IsVREnabled()) {
 		return false;
 	}
```

Comment:
Although fine for now, I think it'll end up making sense to reverse this onto where VR is configured (until VR is available with Vulkan.)

-[Unknown]

Reply:
Yes, this is just a minimal step, how configuration works will be redone later.